src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -4047,7 +4052,7 @@ public static void parseTownMergeCommand(Player player, String[] args) {
 			}			
 		}
 		
-		if (cost > 0) {
+		if (cost > 0 || admin) {

```

Comment:
I dont think this `|| admin` is needed here, I think what you were going for was that admin-initiated merges wouldn't cost anything? What you probably want on line 4043 is `cost = admin ? 0 : baseCost + townblockCost + bankruptcyCost;` then the diff on 4045 can be undone.